wcfsetup/install/files/lib/acp/form/CaptchaQuestionEditForm.class.php

```diff
@@ -23,119 +21,21 @@ class CaptchaQuestionEditForm extends CaptchaQuestionAddForm
      */
     public $activeMenuItem = 'wcf.acp.menu.link.captcha.question.list';
 
-    /**
-     * edited captcha question
-     * @var CaptchaQuestion
-     */
-    public $captchaQuestion;
-
-    /**
-     * id of the edited captcha question
-     * @var int
-     */
-    public $captchaQuestionID = 0;
-
     /**
      * @inheritDoc
      */
-    public function assignVariables()
-    {
-        parent::assignVariables();
-
-        I18nHandler::getInstance()->assignVariables(!empty($_POST));
+    public $formAction = 'edit';
 
-        WCF::getTPL()->assign([
-            'action' => 'edit',
-            'captchaQuestion' => $this->captchaQuestion,
-        ]);
-    }
-
-    /**
-     * @inheritDoc
-     */
-    public function readData()
+    public function readFormParameters()
```

Comment:
This is the wrong method:
```suggestion
    public function readParameters()
```
(also update the parent call)